lavamoat/build-system/policy.json

```diff
@@ -4250,1314 +4250,547 @@
-        "webpack>micromatch>fragment-cache": true,
-        "webpack>micromatch>regex-not": true,
-        "webpack>micromatch>snapdragon": true,
-        "webpack>micromatch>to-regex": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>define-property": {
-      "packages": {
-        "webpack>micromatch>define-property>is-descriptor": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets": {
-      "globals": {
-        "__filename": true
-      },
-      "packages": {
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>debug": true,
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property": true,
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>extend-shallow": true,
-        "webpack>micromatch>extglob>expand-brackets>posix-character-classes": true,
-        "webpack>micromatch>regex-not": true,
-        "webpack>micromatch>snapdragon": true,
-        "webpack>micromatch>to-regex": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>debug": {
-      "builtin": {
-        "fs.SyncWriteStream": true,
-        "net.Socket": true,
-        "tty.WriteStream": true,
-        "tty.isatty": true,
-        "util": true
-      },
-      "globals": {
-        "chrome": true,
-        "console": true,
-        "document": true,
-        "localStorage": true,
-        "navigator": true,
-        "process": true
-      },
-      "packages": {
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>debug>ms": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property": {
-      "packages": {
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor": {
-      "packages": {
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor>is-accessor-descriptor": true,
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor>is-data-descriptor": true,
-        "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor>kind-of": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor>is-accessor-descriptor": {
-      "packages": {
-        "gulp-watch>anymatch>micromatch>kind-of": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>define-property>is-descriptor>is-data-descriptor": {
-      "packages": {
-        "gulp-watch>anymatch>micromatch>kind-of": true
-      }
-    },
-    "gulp-watch>chokidar>readdirp>micromatch>extglob>expand-brackets>extend-shallow": {
-      "packages": {
```

Comment:
We really gotta figure this out... everyone's systems are producing different policies.

Reply:
@Gudahtt could you try wiping node_modules and regenerating to see if the diff lessens?

Reply:
Yeah It's not good. I tried restoring this via the policy override, but adding `fsevents` didn't trigger it to fill out the `fsevents` dependencies. So it wasn't really viable, we'd have to copy every deleted line here into the override to get it to stick.

Reply:
I tried wiping out `node_modules`, adding `gulp-watch>chokidar>fsevents` to the override (it's the source of most of these removals), then reinstalling and re-running `lavamoat:build:auto`. No dice.

Reply:
I noticed in one of my new prs that these are all removed as well, it doesn't even seem to be deterministic on the same machine with node_modules fresh. Oh well, a problem for a different PR